.github/workflows/pytest.yml

```diff
@@ -41,14 +41,23 @@ jobs:
             ${{ runner.os }}-pip-
 
       - name: pip installs
-        run: |
-          pip install pytest pytest-cov pytest-xdist setuptools_scm
-          pip install -e .
+        run: pip install -e .[test]
+
       - name: run pytest
         run: |
-          pytest --runslow -n2
+          pytest -n 2 --runslow tests/
+          echo "Exited with '$?'"
+
+      - name: run pytest on notebooks (append coverage)
+        if: (matrix.os == 'ubuntu-latest') && (matrix.py == '3.8')
```

Comment:
I just wonder if one could write the following in GH-actions:
```

if: (matrix.os == ...)
   - name: run pytest on notebooks....
   - run: ...
   - name: ... 
```

Reply:
pretty sure that's not possible because it would conditionally change the workflow setup (would be nice though)